hedera-node/src/test/java/com/hedera/services/txns/crypto/CryptoTransferTransitionLogicTest.java

```diff
@@ -262,10 +254,7 @@ void translatesUnknownException() {
 		givenValidTxnCtx(withAdjustments(a, -2L, b, 1L, c, 1L));
 
 		// when:
-		subject.doStateTransition();
-
-		// then:
-		verify(txnCtx).setStatus(FAIL_INVALID);
+		assertThrows(Exception.class, () -> subject.doStateTransition());
```

Comment:
and here

Reply:
Regarding this - I've checked it carefully, and we no longer need this test. Reasons:
* `public boolean tryTransition(@NotNull TxnAccessor accessor)` - this is the signature the function, which gives us the accessor and the test gives us a null `TxnAccessor`
* Exceptions are handled with validation in the refactored transitions.